quarkus/metadata.csv

```diff
@@ -0,0 +1,51 @@
+quarkus.http_server.bytes_written.max,gauge,,byte,,Current maximum number of bytes sent by the server.,0,quarkus,,,
+quarkus.http_server.bytes_written.sum,count,,byte,,Total number of bytes sent by the server.,0,quarkus,,,
+quarkus.http_server.connections.seconds.max,gauge,,second,,The duration of the connections in seconds.,0,quarkus,,,
+quarkus.http_server.requests.seconds.count,count,,,,The number of requests observed so far.,0,quarkus,,,
+quarkus.http_server.requests.seconds.max,gauge,,second,,The current longest request duration in seconds.,0,quarkus,,,
+quarkus.http_server.requests.seconds.sum,count,,second,,Total number of seconds that all requests took so far.,0,quarkus,,,
+quarkus.jvm.buffer.count_buffers,gauge,,buffer,,An estimate of the number of buffers in the pool.,0,quarkus,,,
+quarkus.jvm.buffer.memory_used.bytes,gauge,,byte,,An estimate of the memory that the Java virtual machine is using for this buffer pool.,0,quarkus,,,
+quarkus.jvm.buffer.total_capacity.bytes,gauge,,byte,,An estimate of the total capacity of the buffers in this pool.,0,quarkus,,,
+quarkus.jvm.classes.loaded_classes,gauge,,,,The number of classes that are currently loaded in the Java virtual machine.,0,quarkus,,,
+quarkus.jvm.gc.live_data_size.bytes,gauge,,byte,,Size of long-lived heap memory pool after reclamation.,0,quarkus,,,
+quarkus.jvm.gc.max_data_size.bytes,gauge,,byte,,Max size of long-lived heap memory pool.,0,quarkus,,,
+quarkus.jvm.gc.overhead,gauge,,,,"An approximation of the percent of CPU time used by GC activities over the last lookback period or since monitoring began, whichever is shorter, in the range [0..1].",0,quarkus,,,
+quarkus.jvm.memory.committed.bytes,gauge,,byte,,The amount of memory in bytes that is committed for the Java virtual machine to use.,0,quarkus,,,
+quarkus.jvm.memory.max.bytes,gauge,,byte,,The maximum amount of memory in bytes that can be used for memory management.,0,quarkus,,,
+quarkus.jvm.memory.usage_after_gc,gauge,,percent,,"The percentage of long-lived heap pool used after the last GC event, in the range [0..1].",0,quarkus,,,
+quarkus.jvm.memory.used.bytes,gauge,,byte,,The amount of used memory.,0,quarkus,,,
+quarkus.jvm.threads.daemon_threads,gauge,,thread,,The current number of live daemon threads.,0,quarkus,,,
+quarkus.jvm.threads.live_threads,gauge,,thread,,The current number of live threads including both daemon and non-daemon threads.,0,quarkus,,,
+quarkus.jvm.threads.peak_threads,gauge,,thread,,The peak live thread count since the Java virtual machine started or peak was reset.,0,quarkus,,,
+quarkus.jvm.threads.states_threads,gauge,,thread,,The current number of threads.,0,quarkus,,,
+quarkus.netty.allocator.memory.pinned,gauge,,,,"Size, in bytes, of the memory that the allocated buffer uses.",0,quarkus,,,
+quarkus.netty.allocator.memory.used,gauge,,,,"Size, in bytes, of the memory that the allocator uses.",0,quarkus,,,
```

Comment:
Nit. Byte as units maybe?

Reply:
oh yea, totally missed the "in bytes" part, good catch!